### ISSUE TITLE
508-defect-1 ❗️ Launchblocker: Missing links on Health Services

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21537]
Label: 20133

    - [508-defect-1 :exclamation: Launchblocker]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1

Issue Title: 508 defect 1: all use cases, Health Services missing links
 ## Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a user of VA.gov, I expect that all links work properly.
<h2>Details</h2>
On the Health Services page, two items which promise to be links are not:
<ul>
<li>Learn more about Pharmacy under the pharmacy accordian</li>
<li>The phone number under suicide prevention</li>
</ul>
Acceptance Criteria

Environment

Steps to Recreate

<h2>Proposed Solution (if known)</h2>
<ul>
<li>Link "Learn more about pharmacy" to the page offering more information about that topic.</li>
<li> Link the number to contact under suicide prevention such that a smart phone user can call it.</li>
</ul>
WCAG or Vendor Guidance (optional)

<h2>Screenshots or Trace Logs</h2>

![image](https://user-images.githubusercontent.com/78870095/111204210-27ceb400-8583-11eb-8b6c-1e5936d10ab8.png)

**Comment by 1Copenut** [comment]
@AngelaFowler82 I agree this is a severity 1 issue because of the potential human impact. The Pharmacy does say it has a phone number but not one listed, and the Suicide Prevention is text only. These should both be links.
